frontend/src/streamDetails/cards/StreamLogCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Grid } from "@material-ui/core";
+
+import DashboardCard from "../../general/dashboard/DashboardCard";
+import StreamLogsTableWrapper from "../StreamLogsTableWrapper";
+import * as streamLogsDataSource from "../../api/LogApi";
+
+export default function StreamLogCard(props) {
+  const { streamId } = props;
+
+  return (
+    <DashboardCard title="Logs">
+      <Grid container justify="center" direction="row" spacing={3}>
```

Comment:
DashboardCards already have built-in a GridContainer component. you can remove this

Reply:
77d9018

Reply:
77d9018 2d1e625